extensions/notebook/src/book/bookVersionHandler.ts

```diff
@@ -89,7 +89,7 @@ export function convertTo(version: string, section: JupyterBookSection): Jupyter
 			temp.external = section.external;
 			temp.sections = [];
 			for (let s of section.sections) {
-				const child = this.convertTo(version, s);
+				const child = convertTo(version, s);
```

Comment:
What is this doing now?

Reply:
So actually this is undefined, not sure why it was working before.